common/interfaces/desktop_app_interface/usb_event.c

```diff
@@ -277,8 +291,17 @@ bool usb_get_event(usb_event_t *evt) {
         populate_version_list(&resp);
         send_app_version_list_to_host(&resp);
         reset_event_obj(&usb_event);
+
+      } else if (CORE_MSG_SESSION_START_TAG == request_type) {
+        core_session_parse_message(&core_msg_p);
```

Comment:
```suggestion
        core_session_start_parse_message(&core_msg_p);
```
the function name is too generic, though its parsing only session start queries